examples/object_detector_training/train_yolo_habcam.sh

```diff
@@ -11,4 +11,5 @@ source ${VIAME_INSTALL}/setup_viame.sh
 viame_train_detector \
   -i training_data_habcam \
   -c ${VIAME_INSTALL}/configs/pipelines/train_yolo_704.habcam.conf \
+  -p ${VIAME_INSTALL}/configs/pipelines/stereo_pipeline.pipe \
```

Comment:
Looks like you reference a pipeline not git added to the repo, also I would rename this pipeline to reduce ambiguity, call it training_split_and_stereo.pipe or something similar.

Reply:
Done.